editor/property_editor.cpp

```diff
@@ -551,6 +551,7 @@ bool CustomPropertyEditor::edit(Object *p_owner, const String &p_name, Variant::
 
 				text_edit->show();
 				text_edit->set_text(v);
+				text_edit->select(0, 0, 0, 0);
```

Comment:
Doesn't `text_edit->deselect();` suffice?
Also, the commit log should be edited to reflect what it now does.

Reply:
Done, sorry about that. Also, I wasn't even aware of that function, as select_all() is available to GDScript, shouldn't this be as well?